tests/director/tests/application_api_test.go

```diff
@@ -431,47 +431,139 @@ func TestCreateApplicationWithNonExistentIntegrationSystem(t *testing.T) {
 }
 
 func TestUpdateApplication(t *testing.T) {
```

Comment:
do we need e2e test for the case where auto-discovered S/4 systems have `ppmsProductVersionId` which is less than the `ppmsProductVersionId` corresponding to S/4 HANA Cloud 2208 -> so we should not create ORD webhook? What do you think?

Reply:
I think we have covered that in the unit tests, so probably threre's no need for an e2e test